helm/mds/templates/service.yaml

```diff
@@ -1,17 +1,17 @@
-{{- range $key, $api := .Values.apis }}
+{{- range $name, $api := .Values.apis }}
 {{- if $api.enabled }}
 ---
 apiVersion: v1
 kind: Service
 metadata:
-  name: {{ $key }}
+  name: {{ $name }}
   namespace: {{ $.Release.Namespace }}
 spec:
   selector:
-    app: {{ $key }}
+    app: {{ $name }}
   ports:
   -
-    name: http-{{ $key }}
-    port: {{ $api.port }}
+    name: http-{{ $name }}
+    port: 4000
```

Comment:
I might be reading this wrong, but it looks like we hardcode _all_ services to run on port 4000? That seems a bit odd.

Reply:
Exactly - running on different ports is necessary when running node directly on a local system, but isn't in the context of containerization